test/functional/desired.js

```diff
@@ -33,7 +33,8 @@ const SHOW_XCODE_LOG = checkFeatureInEnv('SHOW_XCODE_LOG');
 const APPS = {
   uiCatalogApp: path.resolve(__dirname, '..', 'assets',
     `${IS_ABOVE_IOS13 ? 'UIKitCatalog' : 'UICatalog'}-iphonesimulator.app`),
-  iosTestApp: path.resolve(__dirname, '..', 'assets', 'TestApp-iphonesimulator.app')
+  iosTestApp: path.resolve(__dirname, '..', 'assets', 'TestApp-iphonesimulator.app'),
```

Comment:
maybe it would make sense to mention the repository address for each test app, so one could easily update/modify it when necessary

Reply:
I see, addressed as https://github.com/appium/appium-xcuitest-driver/pull/1482/commits/8216193c4de4786557959c2b0a39588c62f9eb69